src/lib/markbind/src/lib/markdown-it/markdown-it-icons.js

```diff
@@ -1,15 +1,20 @@
 const octicons = require('@primer/octicons');
 
 module.exports = require('markdown-it-regexp')(
-    /:(fa[brs]|glyphicon|octicon)-([a-z-]+):/,
+    /:(fa[brs]|glyphicon|octicon|octiconlight)-([a-z-]+)(~[a-z-]+)?:/,
     (match, utils) => {
         let iconFontType = match[1];
         let iconFontName = match[2];
+        let iconClass = match[3];
 
         if (iconFontType === 'glyphicon') {
             return `<span aria-hidden="true" class="glyphicon glyphicon-${iconFontName}"></span>`;
         } else if (iconFontType === 'octicon') {
-            return octicons[iconFontName].toSVG();
+            return iconClass ? octicons[iconFontName].toSVG({"class": iconClass})
+                             : octicons[iconFontName].toSVG();
```

Comment:
Should we follow the normal ternary style in other places instead?

.......
  ? ...
   : ...

Reply:
dosen't seem to be resolved @le0tan; The reason this works here is that the `markdown-it` folder is ignored in eslint.
Its a very minor nit though, could resolve before merging if you have the time!

Looks good otherwise 👍